### ISSUE TITLE
Ignore `.docker-root` when downloading workspace

### ISSUE THREAD

**Issue by themysticsavages** [gitpod-io/gitpod#16217]
Downloading a workspace from gitpod.io includes the `.docker-root` folder which is unnecessary in most cases. Could this be excluded from the downloads by any chance?

**Comment by axonasif** [comment]
Hi @themysticsavages, thank you for rising this.

For you it may have been `.docker-root`, which was big. There can be and are more files under `/workspace` that are not strictly related with your git-repo but can contriburte to such massive sizes. So we'd need a more generalized solution.
For context, you had posted this on discord[[1](https://discord.com/channels/816244985187008514/839379835662368768/1071916454510858322)] initially:

<img width="699" alt="Screenshot 2023-02-06 at 11 03 40 PM" src="https://user-images.githubusercontent.com/39482679/217036435-e5c7eb29-6c3b-44d3-9499-84c011865adf.png">

So I assume, having the capability to browse/download workspace files without starting it is desired and likely the _ideal solution_ to this problem.

**Comment by themysticsavages** [comment]
> So I assume, having the capability to browse/download workspace files without starting it is desired and likely the _ideal solution_ to this problem.

Yeah, that is a better fix than excluding .docker-root, I'll update my post to reflect this feature.

**Comment by lemystic** [comment]
bump

**Comment by ajskateboarder** [comment]
bump
